tests/test_cli.py

```diff
@@ -968,3 +970,33 @@ def test_check_workspaces(
         if workflow:
             assert len(result.errors) == 2
             assert any(workflow.workspace_path in str(error) for error in result.errors)
+
+
+def test_quota_set_default_limits_with_user_with_custom_limits(default_user, session):
```

Comment:
You can squash the test commit together with the feature commit. It's nice if features are coming with tests in the same atomic unit. Helps to make sure things work already from the start.

(And while squashing, you may want to restrict the commit log message line length.)

Reply:
Noted and done!